### PR TITLE
Add standalone sequence composition - PMT #109928

### PR DESCRIPTION
**Pull request by nikolas**
The create link is currently only available to superusers.

Requires: https://github.com/ccnmtl/juxtapose/pull/281

### CODE REVIEW

File: mediathread/projects/views.py [hunk 1]
```diff
@@ -390,6 +390,36 @@ class SequenceAssignmentEditView(AssignmentEditView):
...
 
 
+class SequenceEditView(LoggedInCourseMixin, ProjectReadableMixin,
+                       TemplateView):
+
+    template_name = 'projects/sequence.html'
+
+    def get(self, request, *args, **kwargs):
+        """
+        A multi-panel editable view for the specified project
```
Comment (sdreher) [inline mediathread/projects/views.py]: These legacy comments can be removed.

File: mediathread/projects/views.py [hunk 2]
```diff
@@ -390,6 +390,36 @@ class SequenceAssignmentEditView(AssignmentEditView):
...
+
+        Panel 1: Parent Assignment (if applicable)
+        Panel 2: Project
+        Panel 3: Instructor Feedback (if applicable & exists)
+
+        Keyword arguments:
+        project_id -- the model id
+        """
+        project = get_object_or_404(Project, pk=kwargs.get('project_id', None))
+        show_feedback = kwargs.get('feedback', None) == "feedback"
```
Comment (sdreher) [inline mediathread/projects/views.py]: Feedback and the notion of a "response" are applicable only to the assignment structure. These can be removed here.

**Comment by sdreher** [comment]
This looks good overall. The only missing piece I see is the ability to save at various levels: private, public, public to world.

**Comment by nikolas** [comment]
Right, yeah I remember we talked about that step. I will add that next. We want it to only prompt the user for the visibility level the first time they save, right?

**Comment by sdreher** [comment]
No - levels should be available at all times. Akin to composition save.

**Pull request merged**